src/Spd.Manager.Licence/ControllingMemberCrcAppManager.cs

```diff
@@ -56,22 +56,22 @@ public async Task<ControllingMemberCrcAppCommandResponse> Handle(ControllingMemb
         createApp.UploadedDocumentEnums = GetUploadedDocumentEnums(cmd.LicAppFileInfos, new List<LicAppFileInfo>());
 
         var response = await _controllingMemberCrcRepository.CreateControllingMemberCrcApplicationAsync(createApp, ct);
-        
+
         await UploadNewDocsAsync(request.DocumentExpiredInfos, cmd.LicAppFileInfos, response.ControllingMemberCrcAppId, response.ContactId, null, null, null, null, null, ct);
-        decimal cost = await CommitApplicationAsync(request, response.ControllingMemberCrcAppId, ct);
+        //TODO: develop new commit method for cm-crc app
+        //decimal cost = await CommitApplicationAsync(request, response.ControllingMemberCrcAppId, ct);
```

Comment:
Please add the code, or create jira ticket. and add ticket number here.
If we do not commit application, we cannot say we completed new cm app feature.